packages/dd-trace/test/config.spec.js

```diff
@@ -672,7 +672,46 @@ describe('Config', () => {
 
     const config = new Config()
 
-    expect(config.telemetryEnabled).to.be.false
+    expect(config.telemetry.enabled).to.be.false
+  })
+
+  it('should set telemetry default values', () => {
+    const config = new Config()
+
+    expect(config.telemetry).to.not.be.undefined
+    expect(config.telemetry.enabled).to.be.true
+    expect(config.telemetry.logCollection).to.be.true
+    expect(config.telemetry.debug).to.be.false
+  })
+
+  it('should not set DD_TRACE_TELEMETRY_ENABLED', () => {
+    process.env.DD_TRACE_TELEMETRY_ENABLED = 'false'
+
+    const config = new Config()
+
+    expect(config.telemetry.enabled).to.be.false
+
+    delete process.env.DD_TRACE_TELEMETRY_ENABLED
```

Comment:
I think that it is better if you get the value in before the test and restore it after the test instead of assume that it is not defined.